animation_nodes/ui/node_menu.py

```diff
@@ -517,6 +517,9 @@ def draw(self, context):
         insertNode(layout, "an_GPFrameFromStrokesNode", "Frame From Strokes")
         insertNode(layout, "an_GPLayerFromFramesNode", "Layer From Frames")
         insertNode(layout, "an_GPObjectOutputNode", "Object Output")
+        layout.separator()
+        insertNode(layout, "an_GPMaterialOutputNode", "Material Output")
```

Comment:
Also add this to the Materials panel as well.

Reply:
Added.